capstone/src/main/webapp/youtube_analyzer.html

```diff
@@ -69,10 +69,11 @@ <h1>Youtube Analysis</h1>
         <div id="available-attributes"></div>
         <button onclick="callYoutube()" class="btn btn-primary">Submit</button>
         <button onclick="getKeywordSearchResults()" class="btn btn-primary" id="keywordSearch">Keyword Search</button>
-        <button onclick="beginDownload()" class="btn btn-primary" id="keywordSearch">Download Output</button>
       </form><br>
       <div id="comment-list"></div>
        <div id="chart-container"></div>
+       <div id="table-container"></div>
+       <button onclick="beginDownload()" class="btn btn-primary" id="download" disabled="true">Download Output</button>
```

Comment:
nit: Instead of `disabled="true"`, I think you can just put `disabled`

Reply:
addressed in "address chris nits"